modules/ROOT/pages/edifact-receive-read-settings.adoc

```diff
@@ -22,42 +22,52 @@ It can take up to five minutes or so for your configuration changes to sync with
 To configure acknowledgements:
 
 . Enable the *Send CONTRL functional acknowledgement when messages fail to route to a message flow* setting.
-. Select an endpoint where the functional acknowledgement message will be sent to the trading partner.
+. Select an endpoint from which the functional acknowledgement message will be sent to the trading partner.
 
-The *Send CONTRL functional acknowledgement when messages fail to route to a message flow* setting provides a functional acknowledgment (the receipt and syntactical acceptability of a message) with the exchange. Functional acknowledgment settings are configured for each message flow and apply only when there is a failure to route the message to one of the message flows.
+The *Send CONTRL functional acknowledgement when messages fail to route to a message flow* setting provides a functional acknowledgment (the receipt and syntactical acceptability of a message) with the exchange.
+
+Functional acknowledgment settings are configured for each message flow and apply only when there is a failure to route the message to one of the message flows.
 
 == Validation settings
 
-Select one or more of the following options to ensure that all EDIFACT messages received from the trading partner meet validation rules. If a message does not meet an enabled rule, Partner Manager rejects it. Parter Manager logs all validation errors and reports them in the CONTRL acknowledgments.
+Select one or more of the following options to ensure that all EDIFACT messages received from the trading partner meet validation rules. If a message does not meet an enabled rule, Partner Manager rejects it.
```

Comment:
I wonder if the table should follow the first sentence here, followed by the the second and the text in Line 35?

Select one of the following...:
[table]
If a message does not match.... Partner Manager logs....